### DISCUSSION TITLE
Channels that are Frequent Targets for Spam

### DISCUSSION THREAD

**Discussion by blahguy88** [ThioJoe/YT-Spammer-Purge#1112] [General]
We should list channels that are frequently attacked by spammers so we can coordinate scans better. After all, the more people report the spammers, the sooner they get terminated. I'll start with...

- [GTLive](https://www.youtube.com/channel/UCAaegDUlb7doIKo0Rc1F08g)
- [The Game Theorists](https://www.youtube.com/channel/UCo_IB5145EVNcf8hw1Kku7w)
- [The Film Theorists](https://www.youtube.com/channel/UC3sznuotAs2ohg_U__Jzj_Q)
- [The Food Theorists](https://www.youtube.com/channel/UCHYoe8kQ-7Gn9ASOlmI0k6Q)
- [penguinz0](https://www.youtube.com/channel/UCq6VFHwMzcMXbuKyG7SQYIg)
- [GLITCH](https://www.youtube.com/channel/UCn_FAXem2-e3HQvmK-mOH4g)
- [KreekCraft](https://www.youtube.com/channel/UCxsk7hqE_CwZWGEJEkGanbA)
- [Flamingo](https://www.youtube.com/channel/UCm-X6o81nRsXQTmqpyArkBQ)
- [Aphmau](https://www.youtube.com/channel/UCzYfz8uibvnB7Yc1LjePi4g)
- [Dafuq!?Boom!](https://www.youtube.com/channel/UCsSsgPaZ2GSmO6il8Cb5iGA)
- [Vivziepop](https://www.youtube.com/channel/UCzfyYtgvkx5mLy8nlLlayYg)
- [MeatCanyon](https://www.youtube.com/channel/UC91V6D3nkhP89wUb9f_h17g)
- [Parlo](https://www.youtube.com/channel/UCh-W5cTLc5sXxleeLdDmEJQ)
- [Daily Dose of Internet](https://www.youtube.com/channel/UCdC0An4ZPNr_YiFiYoVbwaw)
- [SMG4](https://www.youtube.com/channel/UC8LcA3grYZg0GNpxlXh8owg)
- [Sidemen](https://www.youtube.com/channel/UCDogdKl7t7NHzQ95aEwkdMw)
- [Danno Cal Drawings](https://www.youtube.com/channel/UCCuZ0r9pDMI8DwhtRV1fNhg)
- [Dude Perfect](https://www.youtube.com/channel/UCRijo3ddMTht_IHyNSNXpNQ)
- [DJLovesTurbo](https://www.youtube.com/channel/UCrawS1lshIj94Rhm8N4Y-qA)
- [MrBeast](https://www.youtube.com/channel/UCX6OQ3DkcsbYNE6H8uQQuVA)

**Comment by thecoolmalwaredatabase** [comment]
is was a great idea

**Comment by Casondoesstuuffz** [comment]
And Alan Becker and the official game dev Channels like Minecraft and FeedStunz and LMG (Linus media group)  and  others

**Reply by alphaleadership** [reply]
maybe add an option 
to the tool to report iin this discussion
